src/main/java/org/folio/services/BulkProcessingContext.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.services;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class BulkProcessingContext {
```

Comment:
@RuslanLavrov Let's add documentation for this class for a better understanding of the main purpose of this entity.

Reply:
added